docs/apps/rabbitmq.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: rabbitmq
+title: RabbitMQ
+sidebar_label: RabbitMQ
+---
+
+Kubernetes operator to deploy and manage RabbitMQ clusters.
+
+## Overview
+
+[RabbitMQ](https://github.com/rabbitmq/cluster-operator) is a controller and custom resource definition (CRD) designed for the lifecycle (creation, upgrade, graceful shutdown) of a RabbitMQ cluster.
+
+## Creating RabbitMQ Clusters
```

Comment:
This is very valuable content letting ppl know how to work with this integration.
I am wondering if this part is more applicable to Labs. It may need some polishing and more structure, e.g.:
1. enable RabbitMQ operate  (admin)
2. enabling RabbitMQ cluster
3. adding Queue to RabbitMQ cluster
4. connecting app to RabbitMQ cluster
5. accessing the RabbitMQ cluster dashboard (team)

What do you think?